website/docs/d/networkfirewall_firewall_policy.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+subcategory: "Network Firewall"
+layout: "aws"
+page_title: "AWS: aws_networkfirewall_firewall_policy"
+description: |-
+  Retrieve information about a firewall policy.
+---
+
+# Data Source:  aws_networkfirewall_firewall_policy
+
+Retrieve information about a firewall policy.
+
+## Example Usage
+
+```terraform
+data "aws_networkfirewall_firewall_policy" "example" {
+  arn  = var.firewall_policy_arn
+  name = var.firewall_policy_name
+}
+```
+
+## Argument Reference
+One or more of the following arguments are required:
+
+* `arn` - The Amazon Resource Name (ARN) of the firewall policy.
+* `name` - The descriptive name of the firewall policy.
+
+
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `arn` - The ARN of the firewall policy.
+* `description` - A description of the firewall policy.
+* `firewall_policy` - The policy for the specified firewall policy.
```

Comment:
This is a structure that should be defined in the documentation. Either point readers to how the resource defines it at https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/networkfirewall_firewall_policy, or copy and refactor the wording from the resource's documentation